src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/magnetic.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:magnetic",
+  "text": [
+    {
+      "text": "The force is strong with this tool."
+    }
+  ],
+  "effects": [
+    "Hitting things brings items towards you",
+    "Multiple levels, each level increases magnetic strength",
```

Comment:
Technically, its magnetic speed and duration. That may be more detail than we need in the book.